Jenkinsfile_CNP

```diff
@@ -1,14 +1,12 @@
 #!groovy
 
-@Library("Infrastructure")
-import uk.gov.hmcts.contino.AppPipelineConfig
+@Library("Infrastructure@movehighleveldataseupinpreview")
```

Comment:
`setup` have missing `t` in   `movehighleveldataseupinpreview`, is this right?

Reply:
Its just the jenkins branch name which had typo, u can ignore.